propolis/src/inventory.rs

```diff
@@ -483,258 +494,3 @@ impl From<EntityID> for u64 {
         id.num
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
It seems pretty likely that the hierarchy and/or interfacing for the inventory will continue to change as we get further along with the migration stuff, so I didn't want to burn a bunch of time on keeping tests in lockstep until its future was more certain